modules/mgidXBidAdapter.js

```diff
@@ -4,6 +4,7 @@ import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
 import { config } from '../src/config.js';
 import { USERSYNC_DEFAULT_CONFIG } from '../src/userSync.js';
 import { isBidRequestValid, buildRequestsBase, interpretResponse } from '../libraries/teqblazeUtils/bidderUtils.js';
+import { getUserSyncs } from './mgidBidAdapter.js'
 
```

Comment:
Generally this is looking great! However, you can import from /libraries/..., not from another adapter, so please move this function to a file in that folder for both